hipamd/include/hip/amd_detail/amd_hip_fp16.h

```diff
@@ -530,1250 +530,1247 @@ THE SOFTWARE.
         };
         // END STRUCT __HALF2
 
-        namespace
```

Comment:
why do we need anonymous namespace here before? Is it for avoiding duplication with other definition?

Reply:
Really unsure— looking at the blame there isn’t any context to including the anon namespace unfortunately. Since the function parameters are different from other definitions with the same name we shouldn’t have any duplicate function name issues (thanks to mangling/function overloading)